terra_layer/serializers.py

```diff
@@ -107,3 +109,21 @@ def _update_m2m_through(self, instance, field, serializer):
     class Meta:
         model = Layer
         fields = '__all__'
+
+
+class SceneListSerializer(ModelSerializer):
+    detail_url = SerializerMethodField()
+
+    def get_detail_url(self, obj):
+        return reverse('terralayer:scene-detail', args=[obj.slug])
```

Comment:
You should set the method `get_absolute_url()` on the model which is more Django-standard, and use it as field

Reply:
Done